src/components/try-hcp-callout/components/standalone-link-contents/standalone-link-contents.module.css

```diff
@@ -0,0 +1,31 @@
+.root {
+	align-items: center;
+	color: var(--token-color-foreground-strong);
+	display: flex;
+	font-weight: var(--token-typography-font-weight-medium);
+	gap: 6px;
+	line-height: var(--line-height);
+	margin-bottom: 0;
+	margin-top: 24px;
+
+	& svg {
+		flex-shrink: 0;
+		transition: transform 0.2s;
+
+		@nest :global(a):hover & {
```

Comment:
This is a bit of an interesting one - found this was a neat and probably better alternative to having this component receive an `isHovered` prop or something. Open to other ideas here, of course 👍

Reply:
Interesting idea, we ran into a similar challenge with our [ExpandableArrow](https://react-components.vercel.app/components/expandablearrow) component that needs to be triggered by a parent component. Couldn't guarantee it'd always be within an `a` tag though. For example our link in the card does not wrap the card content similar to how your CardLink works. Ended up adding a prop to trigger the state change.

Reply:
Now that I mentioned it, this might not be working correctly, since the link tag is not wrapping the content?

Reply:
Ah yeah great catch 💖  this needs to be `[class*="card-link"]`, I had that before but then thought I could clean it up 😅 Updating now! 

Great point re: guarantee for the hover behaviour. This is definitely a pattern we should probably explore further. In the context of this PR, and since this is a lil piece scoped for use in this component only, personally I think the relative brittleness might be worth the trade-off.

Reply:
Agreed!

Reply:
Also happy to switch to the `isHovered` hook approach - I'm totes happy either way, very open if anyone has strong or even lukewarm or weak preference there 🙇